library/gradle

```diff
@@ -4,59 +4,59 @@ GitRepo: https://github.com/keeganwitt/docker-gradle.git
 
 # Gradle 7.x
 
-Tags: 7.6.0-jdk8, 7.6-jdk8, 7-jdk8, jdk8, 7.6.0-jdk8-jammy, 7.6-jdk8-jammy, 7-jdk8-jammy, jdk8-jammy
+Tags: 8.0.0-jdk8, 8.0-jdk8, 7-jdk8, jdk8, 8.0.0-jdk8-jammy, 8.0-jdk8-jammy, 7-jdk8-jammy, jdk8-jammy
```

Comment:
I think this should be a new set of tags rather than replacement of all v7 ones. Also, you missed all "7-" tags anyway :slightly_smiling_face:

Reply:
And the comment line above if you're going to drop all v7 tags 😉

Reply:
Yep, I sure did. Thank you.